src/applications/disability-benefits/all-claims/content/toxicExposure.jsx

```diff
@@ -239,6 +242,25 @@ export function validateTEConditions(errors, formData) {
   }
 }
 
+/**
+ * Validates selected Gulf War 1990 locations.
+ * If the 'none' checkbox is selected
+ * along with another location adds an error.
+ *
+ * @param {object} errors - Errors object from rjsf
+ * @param {object} formData
+ */
+export function validateGulfWar1990Locations(errors, formData) {
+  const { gulfWar1990 = {} } = formData;
+
+  if (
+    gulfWar1990.none === true &&
+    Object.values(gulfWar1990).filter(value => value === true).length > 1
+  ) {
+    errors.gulfWar1990.addError(noneAndLocationError);
+  }
+}
```

Comment:
Looks sound so far. Is the intention to keep this strictly as a frontend validation, or will we have to account for a None type being submitted as well?

Reply:
Good question. There will be some backend work needed as well. See notes [here](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/72816)